.gitignore

```diff
@@ -38,3 +38,5 @@ node_modules
 .idea
 **/*.iml
 styleguide.css
+
+package-lock.json
```

Comment:
Isn't the purpose of a package-ock.json to put it into the repo to ensure same install on any machine ?